ddtrace/llmobs/_llmobs.py

```diff
@@ -273,9 +292,39 @@ def annotation_context(
                         This argument is only applicable to LLM spans.
         :param name: Set to override the span name for any spans annotated within the returned context.
         """
-        return AnnotationContext(
-            cls._instance.tracer, lambda span: cls.annotate(span, tags=tags, prompt=prompt, _name=name)
-        )
+        # id to track an annotation for registering / de-registering
+        annotation_id = uuid4().hex
```

Comment:
uuids are quite expensive to generate:

```
In [4]: %timeit uuid.uuid4().hex
1.42 μs ± 21.4 ns per loop (mean ± std. dev. of 7 runs, 1,000,000 loops each)

In [7]: %timeit rand64bits()
23 ns ± 0.231 ns per loop (mean ± std. dev. of 7 runs, 10,000,000 loops each)
```

we should be able to re-use `ddtrace.internal._rand.rand64bits()` here.